ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AwsSecurityHubIntegrationForm.tsx

```diff
@@ -113,6 +121,8 @@ function AwsSecurityHubIntegrationForm({
     initialValues = null,
     isEditable = false,
 }: IntegrationFormProps<AwsSecurityHubIntegration>): ReactElement {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const showSTSCheckbox = isFeatureFlagEnabled('ROX_CLOUD_CREDENTIALS');
```

Comment:
For your information, not a change request, although there are examples of this naming convention in ui code, boolean variables have `is` or `has` prefix when fluent, therefore more like:

```ts
const isCloudCredentialsEnabled = isFeatureFlagEnabled('ROX_CLOUD_CREDENTIALS');
```